src/lib.rs

```diff
@@ -1,18 +1,23 @@
 extern crate rustc_serialize;
+use std::string;
 
 pub enum Value {
   Boolean(bool),
   TinyInt(u8),
+  TinyText(Result<String, string::FromUtf8Error>)
```

Comment:
I'm not really sure of the best Rusty approach.  I think `unpack` should probably be returning a `Result<Value, PackStreamError>` rather than an `Option`